### PR TITLE
add mailchimp

### PR DESCRIPTION
**Pull request by jordanskole**
Add mailchimp

### CODE REVIEW

File: src/icons/mailchimp.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...,12.89,12.4a11.89,11.89,0,0,1,.18,2.85,15.07,15.07,0,0,1-.38,2.83c-1.39,6-7.12,11.8-16.57,10.33-1.74-.25-2.87-.47-3.21,0-.73,1,3.32,5.65,10.74,5.48,7.25-.15,14.45-5.58,16.76-13.75a20.78,20.78,0,0,0,.76-4.49,18.64,18.64,0,0,0-.27-4.77C138.51,320.65,128.1,318.5,124.22,315.41ZM377,240.57c-2.87-.45-5.69,2.38-6.3,6.31s1.21,7.48,4.08,7.93,5.69-2.38,6.3-6.31S379.9,241,377,240.57ZM345.54,252c-1.58,2.88.34,7,4.29,9.12s8.42,1.58,10-1.29-.34-7-4.29-9.12S347.11,249.16,345.54,252Zm-115.4-116.3c54.24-35.16,108.42-39.2,127-24.93h0c0-.09-.33-.92-.45-1.14h0c-4.71-10-15.71-17-25.51-19.8a38.23,38.23,0,0,1,5.21,7,78,78,0,0,0-27.47-11.68s1.22.88,1.43,1.07c2,1.9,4.76,4.91,5.88,7.49-9.8-3.93-22.19-6.14-32.79-4.1L282.1,90s1.38.34,1.69.43c3.52,1,8.54,3.19,10.75,6.12-17.6-3.1-37,.33-47.41,6.17a24.2,24.2,0,0,1,2.47,0c3.84.1,11.58.62,14.83,2.69-11.1,2.26-27.17,7.23-35.82,14.66,1.53-.18,10.2-1.28,13.74-.71-47.61,27.25-69.29,68.44-69.29,68.44h0C187.09,169.44,205.3,151.83,230.14,135.74Z" transform="translate(-12.08 -1)"/><path class="cls-1" d="M358.11,212.55c-.07-3.74.52-9.91,4-11.06h0c6-2.09,14,13.32,14.35,27a30.15,30.15,0,0,0-16.08-2.89A46.31,46.31,0,0,1,358.11,212.55Z" transform="translate(-12.08 -1)"/></svg>
```
Comment (ScionOfBytes) [inline src/icons/mailchimp.svg]: @jordanskole Totally nitpicking, but can you add a newline at the end of the file?

Comment (jordanskole) [inline src/icons/mailchimp.svg]: @ScionOfBytes lol sure

Comment (jordanskole) [inline src/icons/mailchimp.svg]: @ScionOfBytes did this work?

**Comment by jordanskole** [comment]
@AliasIO mind taking a look at this one when you have a few?

**Comment by AliasIO** [comment]
Looks good, thank you.

**Pull request merged**